sdks/java/io/azure/build.gradle

```diff
@@ -29,13 +30,19 @@ ext.summary = "IO library to read and write Azure services from Beam."
 repositories { jcenter() }
 
 dependencies {
+  compile library.java.commons_io
+  permitUnusedDeclared library.java.commons_io
+  compile library.java.slf4j_api
   compile library.java.vendored_guava_26_0_jre
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile "com.azure:azure-storage-blob:12.8.0"
   compile "com.azure:azure-identity:1.0.8"
   compile "com.microsoft.azure:azure-storage:8.6.5"
-  compile "commons-io:commons-io:2.6"
-  compile library.java.slf4j_api
+  compile "com.azure:azure-core:1.6.0"
+  compile "com.azure:azure-storage-common:12.1.0"
```

Comment:
azure-storage-common should have the same version as azure-storage-blob (since azure-storage-common is currently a transitive dependency imported by azure-storage-blob).